db/migrate/20220921093216_create_tickets.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTickets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tickets do |t|
```

Comment:
where do you have a price? 🤔

Reply:
We agreed at DB modeling workshops that it can be a part of seance